src/agenda/index.js

```diff
@@ -92,7 +92,9 @@ export default class AgendaView extends Component {
   }
 
   componentDidUpdate(prevProps) {
-    if (!prevProps.items) {
+    if (this.props.selected && !dateutils.sameDate(parseDate(this.props.selected), parseDate(prevProps.selected))) {
+      this.setState({selectedDay: parseDate(this.props.selected)});
```

Comment:
Are 'this.currentMonth' and 'topDay' shouldn't be updated as well?

Reply:
We don't use those props in out project so we don't know them.
I wouldn't feel confident on doing anything with them as I don't trust I'd be able to properly test their changes.

Reply:
This is a simple pull request, and it is actually fixing a bug. Could we please move this forward so we can have this issue fixed in the main branch? I understand that reviewing and testing PRs takes time, but we are also investing time as collaborators fixing it, and the issue has been open for a long time :)

If I were to review it, I would say everything is ok now.